source/code-snippets/usage-examples/changeStream.js

```diff
@@ -0,0 +1,42 @@
+const { MongoClient } = require("mongodb");
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+let changeStream, interval;
+async function run() {
+  try {
+    await client.connect();
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+    // create a change stream on the movies collection
```

Comment:
Nitpick: I think the terminology you used in the description was "open a change stream" so we should try to stay consistent.

Reply:
Fixed